### ISSUE TITLE
Build pipeline failed when installing python-genpackagedoc

### ISSUE THREAD

**Issue by ngoan1608** [test-fullautomation/RobotFramework_AIO#369]
Hi,

Both build pipelines OSS and BIOS are failed now due to same issue with genpackagedoc:
-Referent OSS pipeline: https://github.com/test-fullautomation/RobotFramework_AIO/actions/runs/11227945384/job/31211024409

```
Chapter 1.
(./Introduction.tex) [1]
Chapter 2.
(./Description.tex (/usr/share/texlive/texmf-dist/tex/latex/psnfss/ot1pcr.fd)
[2]
Underfull \hbox (badness 10000) in paragraph at lines 182--187

[3] [4] [5]
! Undefined control sequence.
l.372 \pandocbounded
                    {\includegraphics[keepaspectratio]{./pictures/TOC01.png}}
? 
! Emergency stop.
l.372 \pandocbounded
                    {\includegraphics[keepaspectratio]{./pictures/TOC01.png}}
!  ==> Fatal error occurred, no output PDF file produced!
Transcript written on GenPackageDoc.log.

LaTeX compiler returned 1


Error: !!! ERROR !!!
[CDocBuilder.Build] : !!! ERROR !!!
[CDocBuilder.__GenDocPDF] : LaTeX compiler not returned expected value 0!
```

Thank you,
Ngoan

**Comment by ngoan1608** [comment]
Hi @HolQue ,
Hi @test-fullautomation ,

I checked and found that the genpackagedoc failed due to new release `pypandoc_binary` version **1.14**.
It was stuck and failed at step to generate pdf file as above information.

When I installed the previous version of `pypandoc_binary` - **1.13**, genpackagedoc is fine again.
I used this fixed version as workaround solution #370  to avoid error in build pipeline while waiting a change (if requires) from genpackagedoc package.

Thank you,
Ngoan
